font-src/glyphs/marks/overlay.ptl

```diff
@@ -200,6 +200,58 @@ glyph-block Mark-Overlay : begin
 			flat (SB - O * 3 + fine - Width) (XH * 0.5 - CAP * 0.6) [widths.center : 2 * fine]
 			curl (RightSB + O * 3 - fine - Width) (XH * 0.5 + CAP * 0.6)
 
+	create-glyph 'longVStrokeOver' 0x20D2 : glyph-proc
+		set-width 0
+		local fine : 0.375 * OverlayStroke
+		set-mark-anchor 'slash' markMiddle (XH * 0.5) markMiddle (XH * 0.5)
+		include : dispiro
+			flat markMiddle (XH * 0.5 - CAP * 0.6) [widths.center : 2 * fine]
+			curl markMiddle (XH * 0.5 + CAP * 0.6)
+
+	create-glyph 'shortVStrokeOver' 0x20D3 : glyph-proc
+		set-width 0
+		local fine : 0.375 * OverlayStroke
+		set-mark-anchor 'slash' markMiddle (XH * 0.5) markMiddle (XH * 0.5)
+		include : dispiro
+			flat markMiddle (XH * (-0.1)) [widths.center : 2 * fine]
+			curl markMiddle (XH * 1.1)
+
+	create-glyph 'revLongSlash' 0x20E5 : glyph-proc
+		set-width 0
+		local fine : 0.375 * OverlayStroke
+		set-mark-anchor 'slash' markMiddle (XH * 0.5) markMiddle (XH * 0.5)
+		include : dispiro
+			flat (RightSB + O * 3 - fine - Width) (XH * 0.5 - CAP * 0.6) [widths.center : 2 * fine]
+			curl (SB - O * 3 + fine - Width) (XH * 0.5 + CAP * 0.6)
+
+	create-glyph 'dblLongVStrokeOver' 0x20E6 : glyph-proc
+		set-width 0
+		local fine : 0.3 * OverlayStroke
```

Comment:
Don't do this: it will make the overlay stroke too thin in Thin weights.
Use another `[AdviceStroke]` call with a larger crowdedness (like 6)